### PR TITLE
Add Keygen to FFI.

### PR DESCRIPTION
**Pull request by nilz3ro**
# About These Changes

* Adds `keygen::generate_files` to FFI.
* Adds tests.
* closes casper-network/casper-node#1573

### CODE REVIEW

**Review by darthsiroftardis — APPROVED** [review]

**Review by dwerner — APPROVED** [review]
Looks good!

**Comment by nilz3ro** [comment]
bors merge

File: client/CHANGELOG.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ All notable changes to this project will be documented in this file.  The format
 * Change `put-deploy` and `make-deploy` subcommands to support transfers.
 * Change `make-deploy`, `make-transfer` and `sign-deploy` to not overwrite files unless `--force` is passed.
 * Change `make-deploy`, `make-transfer` and `sign-deploy` to use transactional file writing for enhanced safety and reliability.
-
+* Add `keygen::generate_files` to FFI.
```
Comment (Fraser999) [inline client/CHANGELOG.md:24]: This should be part of the `### Added` section above.

**Review by Fraser999 — APPROVED** [review]
One tiny nitpick, otherwise LGTM.

File: client/CHANGELOG.md (hunk 1 above)
Comment (nilz3ro) [reply client/CHANGELOG.md:24]: 💯

**Comment by nilz3ro** [comment]
bors merge

**Comment by nilz3ro** [comment]
bors merge

**Pull request merged**